src/EPR.Calculator.API/Builder/Summary/Common/CalcResultSummaryUtil.cs

```diff
@@ -458,35 +418,45 @@ public static void SetHeaders(CalcResultSummary result, List<MaterialDetail> mat
 
     public static List<CalcResultSummaryHeader> GetProducerDisposalFeesHeaders()
     {
```

Comment:
Please return IEnumerable<CalcResultSummaryHeader>

Reply:
I have applied the fix, thank you.